packages/main/src/StepInput.js

```diff
@@ -0,0 +1,540 @@
+		 * @public
+		 */
+		min: {
+			type: Float,
+		},
+
+		/**
+		 * Defines a maximum value of the <code>ui5-step-input</code>.
+		 *
+		 * @type {Float}
+		 * @public
+		 */
+		max: {
+			type: Float,
+		},
+
+		/**
+		 * Defines a step of increasing/decreasing the value of the <code>ui5-step-input</code>.
+		 *
+		 * @type {Float}
+		 * @defaultvalue 1
+		 * @public
+		 */
+		step: {
+			type: Float,
+			defaultValue: 1,
+		},
+
+		/**
+		 * Defines the value state of the <code>ui5-step-input</code>.
+		 * <br><br>
+		 * Available options are:
+		 * <ul>
+		 * <li><code>None</code></li>
+		 * <li><code>Error</code></li>
+		 * <li><code>Warning</code></li>
+		 * <li><code>Success</code></li>
+		 * <li><code>Information</code></li>
+		 * </ul>
+		 *
+		 * @type {ValueState}
+		 * @defaultvalue "None"
+		 * @public
+		 */
+		valueState: {
+			type: ValueState,
+			defaultValue: ValueState.None,
+		},
+
+		/**
+		 * Defines whether the <code>ui5-step-input</code> is required.
+		 *
+		 * @since 1.0.0-rc.9
+		 * @type {Boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		required: {
+			type: Boolean,
+		},
+
+		/**
+		 * Determines whether the <code>ui5-step-input</code> is displayed as disabled.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disabled: {
+			type: Boolean,
+		},
+
+		/**
+		 * Determines whether the <code>ui5-step-input</code> is displayed as read-only.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		readonly: {
+			type: Boolean,
+		},
+
+		/**
+		 * Defines a short hint, intended to aid the user with data entry when the
+		 * <code>ui5-step-input</code> has no value.
+		 *
+		 * <br><br>
+		 * <b>Note:</b> When no placeholder is set, the format pattern is displayed as a placeholder.
+		 * Passing an empty string as the value of this property will make the <code>ui5-step-input</code> appear empty - without placeholder or format pattern.
+		 *
+		 * @type {string}
+		 * @defaultvalue undefined
+		 * @public
+		 */
+		placeholder: {
+			type: String,
+			defaultValue: undefined,
+		},
+
+		/**
+		 * Determines the name with which the <code>ui5-step-input</code> will be submitted in an HTML form.
+		 *
+		 * <br><br>
+		 * <b>Important:</b> For the <code>name</code> property to have effect, you must add the following import to your project:
+		 * <code>import "@ui5/webcomponents/dist/features/InputElementsFormSupport.js";</code>
+		 *
+		 * <br><br>
+		 * <b>Note:</b> When set, a native <code>input</code> HTML element
+		 * will be created inside the <code>ui5-step-input</code> so that it can be submitted as
+		 * part of an HTML form. Do not use this property unless you need to submit a form.
+		 *
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		name: {
+			type: String,
+		},
+
+		/**
+		 * Determines the text alignment of the <code>ui5-step-input</code>.
+		 *
+		 * @type {string}
+		 * @defaultvalue "left"
+		 * @public
+		 */
+		textAlign: {
```

Comment:
This seems like the css "textAlign", is it possible to make this work through the CSS only:

```css
ui5-step-input {
   text-align: start|end|center
}

Reply:
Removed